### ISSUE TITLE
Replace Makefile with poe tasks in pyproject.toml

### ISSUE THREAD

**Issue by nat-n** [Diapolo10/python-poetry-template#177]
Hi, GitHub randomly showed me this project which looks very useful.

Firstly, full disclosure: I'm the author of [Poe the Poet](https://poethepoet.natn.io/). I created it because I saw the need for a more delightful alternative to using Makefiles in poetry projects. I would highlight the following advantages over make:
- simpler task declaration within the pyproject.toml without any boilerplate or peculiar syntax (.PHONY?)
- tasks are self documenting
- automatically runs tasks with poetry's venv (no need to specify poetry run) and can be installed directly into poetry as a plugin
- lots of features including much better support for defining task argument 

It has significant adoption: [![PyPI version](https://img.shields.io/pypi/dm/poethepoet.svg)](https://pypistats.org/packages/poethepoet)

Maybe you like just like Makefiles, and that's fair, but if you're open to it I'd be happy to propose a PR to demonstrate what replacing the Makefile with poe tasks would look like in practice 🙂

**Comment by Diapolo10** [comment]
Yes, I recognise you. I believe I've had a chat with you before in the issues of another project, some years ago.

For the time being I'm not particularly interested, and the `Makefile` is mostly there just in case I happen to be on a Linux system and find it more convenient (honestly it's probably outdated anyway, I rarely touch it). Feel free to fork the project and add your modifications though, perhaps you can convince me otherwise.

**Comment by Diapolo10** [comment]
On a mostly unrelated note, I've currently focused on getting proper logging configuration working in the template (in a form that can be easily removed, such as if working on a library), if you happen to have time on your hands feel free to take a look at that. I've been a bit too busy with work lately to really work on free time projects.
